gcis.ttl

```diff
@@ -1033,3 +1033,9 @@ gcis:isSupportedBy  a owl:ObjectProperty ;
 	rdfs:label "Is Supported By" ;
 	rdfs:comment "A finding is supported by data, models, model runs, software, etc.";
 	rdfs:domain gcis:Finding .	
+
+gcis:computingEnvironmentsUsed a owl:DatatypeProperty ;
+	rdfs:label "computing environments used" ;
```

Comment:
It is conventional in RDF for rdfs:labels on properties to be lower-case.  I believe this is a case where the convention GCIS was using is not standard.